jax/random.py

```diff
@@ -120,26 +120,59 @@
 selected with the optional `impl` keyword argument to
 `jax.random.key`. When no `impl` option is passed to the `key`
 constructor, the implementation is determined by the global
-`jax_default_prng_impl` configuration flag.
+`jax_default_prng_impl` configuration flag. The string names of
```

Comment:
Pre-existing issue, but we need double back-ticks for code font in RST (single back-ticks are rendered as itallic)